src/arrays.js

```diff
@@ -1,61 +1,80 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const adjustIndex = index % array.length;
+  return array[adjustIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  const newArray = array.concat(element);
+  return newArray;
```

Comment:
can just do `return array.concat(element)` instead of assigning to a variable first